teitelbot/response/reply/service/email_generator.py

```diff
@@ -0,0 +1,13 @@
+from util.string_utils import translate_to_heb, clean_punctuation
+
+
+def generate(full_name_in_hebrew: str) -> str:
+    name = _get_translated_name(full_name_in_hebrew)
+    suffix = '@g.jct.ac.il'
```

Comment:
We should consider changing this

Reply:
to what suffix?
should we pass it passed via config?

Reply:
We should discuss this fully at the next weekly

Reply:
new issue #33